backends/gdx-backend-headless/src/com/badlogic/gdx/backends/headless/mock/input/MockInput.java

```diff
@@ -41,6 +41,24 @@ public float getAccelerometerY() {
 	public float getAccelerometerZ() {
 		return 0;
 	}
+	
+	@Override
+	public float getGyroscopeX() {
+		// TODO Auto-generated method stub
+		return 0;
+	}
+
+	@Override
+	public float getGyroscopeY() {
+		// TODO Auto-generated method stub
+		return 0;
+	}
+
+	@Override
+	public float getGyroscopeZ() {
+		// TODO Auto-generated method stub
```

Comment:
I'm happy to take them out.  While this implementation concentrates on android, I'm not familiar enough with other features/platforms to properly implement them..   Like MockInput.  I just don't know what it does.